fcrepo-import-export-driver/src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -37,78 +49,145 @@
         importExportOption.setRequired(true);
         importExportOption.setArgs(1);
         importExportOption.setArgName("mode");
-        cmdOptions.addOption(importExportOption);
+        configOptions.addOption(importExportOption);
 
         // Resource option
         final Option resourceOption = new Option("r", "resource", true, "Resource (URI) to import/export");
         resourceOption.setRequired(true);
         resourceOption.setArgs(1);
         resourceOption.setArgName("resource");
-        cmdOptions.addOption(resourceOption);
+        configOptions.addOption(resourceOption);
 
         // Binary Directory option
         final Option binDirOption = new Option("b", "binDir", true, "Directory where binaries (files) are stored");
         binDirOption.setRequired(false);
         binDirOption.setArgs(1);
         binDirOption.setArgName("binDir");
-        cmdOptions.addOption(binDirOption);
+        configOptions.addOption(binDirOption);
 
         // Description Directory option
         final Option descDirOption = new Option("d", "descDir", true, "Directory where RDF descriptions are stored");
         descDirOption.setRequired(true);
         descDirOption.setArgs(1);
         descDirOption.setArgName("descDir");
-        cmdOptions.addOption(descDirOption);
+        configOptions.addOption(descDirOption);
 
         // RDF extension option
         final Option rdfExtOption = new Option("x", "rdfExt", true, "RDF filename extension (default: " +
                 DEFAULT_RDF_EXT);
         rdfExtOption.setRequired(false);
         rdfExtOption.setArgs(1);
         rdfExtOption.setArgName("rdfExt");
-        cmdOptions.addOption(rdfExtOption);
+        configOptions.addOption(rdfExtOption);
 
         // RDF language option
         final Option rdfLangOption = new Option("l", "rdfLang", true, "RDF language (default: " + DEFAULT_RDF_LANG);
         rdfLangOption.setRequired(false);
         rdfLangOption.setArgs(1);
         rdfLangOption.setArgName("rdfLang");
-        cmdOptions.addOption(rdfLangOption);
+        configOptions.addOption(rdfLangOption);
+
+        // Config file option
+        final Option configFileOption = new Option("c", "config", true, "Path to config file");
+        configFileOption.setRequired(true);
+        configFileOption.setArgs(1);
+        configFileOption.setArgName("config");
+        configFileOptions.addOption(configFileOption);
     }
 
-    Config parseConfiguration(final String[] args) {
-        final Config config = new Config();
+    protected Config parseConfiguration(final String[] args) {
+        // Inspect Config File option
+        Config config = parseConfigFileOptions(args);
+        if (config == null) {
+            // Inspect standard command-line options
+            config = parseConfigurationArgs(args);
+        }
+
+        // Write configuration to disk
+        saveConfig(args);
```

Comment:
Should this save the options as parsed in `config` instead of the original command-line params?  Wouldn't this lead to saving the pointer to the user-specified config file?